lib/matplotlib/tests/test_axes.py

```diff
@@ -5149,3 +5149,21 @@ def test_twinx_knows_limits():
     ax2.plot([0, 0.5], [1, 2])
 
     assert((xtwin.viewLim.intervalx == ax2.viewLim.intervalx).all())
+
+
+@image_comparison(baseline_images=['zoomed_axes',
+                                   'inverted_zoomed_axes'],
+                  extensions=['png'])
+def test_zooming_with_inverted_axes():
+    fig, ax = plt.subplots()
+    ax.plot([1, 2, 3], [1, 2, 3])
+    ax.axis([1, 3, 1, 3])
+    from mpl_toolkits.axes_grid1.inset_locator import zoomed_inset_axes
```

Comment:
If it uses `mpl_toolkits`, it doesn't belong in this file. It should go somewhere in `lib/mpl_toolkits/tests`.